packages/block-editor/src/components/block-settings-menu/block-mode-toggle.js

```diff
@@ -36,10 +36,12 @@ export default compose( [
 	withSelect( ( select, { clientId } ) => {
 		const { getBlock, getBlockMode } = select( 'core/block-editor' );
 		const block = getBlock( clientId );
+		const isCodeEditingEnabled = select( 'core/editor' ).getEditorSettings().codeEditingEnabled;
```

Comment:
We can't use `core/editor` in the block editor module. This makes this component not usable outside the post editor (widget editor, standalone editor...).

The fix is to "proxy" this config into the block editor settings (in the `EditorProvider` component) and replace this call with `select( 'core/block-editor' ).getEditorSettings()`

Anyone willing to take care of this before the next release?